documentation/ame-guide/modules/ROOT/pages/edit-models.adoc

```diff
@@ -52,18 +54,20 @@ image::adding-elements.png[width=100%]
 Drag and drop one of the available elements from the xref:ui-overview.adoc#workspace-sidebar[Workspace sidebar].
 You have the option to create new elements or to use already xref::use-external-references.adoc#add-external-references[existing elements] from other namespaces.
 
-NOTE: In order to prevent formatting issues, connect a new element right after you added it. See also xref:known-issues.adoc[known issues].
-
+NOTE: In order to prevent formatting issues, connect a new element right after you added it.
```

Comment:
In order to prevent formatting issues, it is a best practices to always connect a new element with another element directly after adding it.